stats/influxdb/collector.go

```diff
@@ -132,8 +133,39 @@ func (c *Collector) commit() {
 }
 
 func (c *Collector) extractTagsToValues(tags map[string]string, values map[string]interface{}) map[string]interface{} {
+tags:
 	for _, tag := range c.Config.TagsAsFields {
 		if val, ok := tags[tag]; ok {
+			for _, t := range c.Config.IntFields {
+				if tag == t {
+					i, err := strconv.ParseInt(val, 10, 64)
+					if err == nil {
+						values[tag] = i
+						delete(tags, tag)
+						continue tags
```

Comment:
If I understand this correctly this means that if you have tag `shouldBeNumber` and it has value "2" and a value "five", and is in IntFields ...  we will in send `2` (as the integer to InfluxDB)  and the other one as `"five"` how will InfluxDB handle this? 
Also maybe we should report this ... somehow .. a warning in the log?

Reply:
On InfluxDB, it is possible to store a single data point with the `shouldBeNumber` tag =(string) "2" and the `shouldBeNumber` field = (int) 2.  And we can explicitly specify the tag with `shouldBeNumber::tag` in InfluxQL.  

However, practically having tags and fields with the same names are very confusing and I think we'd better document that users are recommended to use a new database if they want to change certain set of tags to fields or vice versa.